index.js

```diff
@@ -22,6 +22,35 @@ const TREZOR_CONNECT_MANIFEST = {
   appUrl: 'https://metamask.io',
 };
 
+const oneKeySpecialVersion = 99;
+const oneKeyVendor = 'onekey.so';
+/**
+ * Distinguish the OneKey hardware wallet by the serialNo prefix
+ * @param {*} features
+ * @returns {'mini' | 'touch' | 'classic' | 'trezor'}
+ */
+function isOneKeyDevice(features) {
```

Comment:
Could we make the semantics more generic?
This could be renamed to `getVendor`

Reply:
To make it more generic and semantic, I added a `__setVendor` instance method that hides all the details of setting the vendor.